app/models/notifications/mentioned_in_comment.rb

```diff
@@ -0,0 +1,27 @@
+module Notifications
+  class MentionedInComment < Mentioned
+    def popup_translation_key
+      "notifications.mentioned_in_comment"
+    end
+
+    def deleted_translation_key
+      "notifications.mentioned_in_comment_deleted"
+    end
+
+    def self.notify(comment, recipient_user_ids)
+      # recipient_user_ids are used in Notifications::Mentioned.notify in order to filter notification
+      # recipients for private posts. It's not clear, whether this is required, since mentions filtering
+      # is performed on the post/comment creation stage and we could consider every mention object as a
+      # sufficient reason for sending a notification. The issue is that recipient_user_ids never
+      # contains parent owner id and therefore the post author doesn't receive the notification if
+      # mentioned in comment.
+      #
+      # I left intact the Notifications::Mentioned.notify and didn't investigate the message dispatcher
+      # but here I add parent_owner_id to the list so the filtering passes well.
```

Comment:
This doesn't work, because private posts are received multiple times (to create the ShareVisibilities), so the mention-notification is only sent, if the user is in the `recipient_ids` (the user who received the visibility). But comments are only received once (all others are ignored), because all visibilities are already there. See `AlsoCommented`.